src/Umbraco.Infrastructure.PublishedCache/PublishedCacheBase.cs

```diff
@@ -10,6 +10,13 @@ namespace Umbraco.Web.PublishedCache
 {
     abstract class PublishedCacheBase : IPublishedCache
     {
```

Comment:
This class can exist in abstractions, it doesn't need to exist in the PublishedCache project, it has no ties to NuCache and could be used by other implementations